test/extended/storage/csi/scsi_overflow.go

```diff
@@ -0,0 +1,202 @@
+		until := time.Now().Add(timeout)
+
+		g.By("Selecting a schedulable node")
+		node, err := node2.GetRandomReadySchedulableNode(ctx, f.ClientSet)
+		e2e.ExpectNoError(err, "getting a schedulable node")
+
+		g.By("Creating a StorageClass")
+		config := driver.PrepareTest(ctx, f)
+		sc, err := createSC(ctx, f, driver, config)
+		e2e.ExpectNoError(err, "creating StorageClass")
+		g.DeferCleanup(func(ctx context.Context) {
+			e2e.Logf("Cleaning up StorageClass %s", sc.Name)
+			err := f.ClientSet.StorageV1().StorageClasses().Delete(ctx, sc.Name, metav1.DeleteOptions{})
+			e2e.ExpectNoError(err, "deleting StorageClass", sc.Name)
+		})
+
+		podCount := csiSuite.lunStressTestConfig.PodsTotal
+		e2e.Logf("Starting %d pods", podCount)
+		for i := 0; i < podCount; i++ {
+			startTestPod(ctx, f, node.Name, config, sc.Name, i)
+		}
+		e2e.Logf("All pods created, waiting for them to start until %s", until.String())
+
+		err = waitForPodsComplete(ctx, f, podCount, timeout)
+		e2e.ExpectNoError(err, "waiting for pods to complete")
+		e2e.Logf("All pods completed, cleaning up")
+	})
+}
+
+// Create one PVC + Pod. Do not wait for the pod to start!
+func startTestPod(ctx context.Context, f *e2e.Framework, nodeName string, config *storageframework.PerTestConfig, scName string, podNumber int) {
+	pvcName := fmt.Sprintf("pvc-%d", podNumber)
+
+	claimSize := config.Driver.GetDriverInfo().SupportedSizeRange.Min
+	if claimSize == "" {
+		claimSize = "1Gi"
+	}
+	claimQuantity, err := resource2.ParseQuantity(claimSize)
+	e2e.ExpectNoError(err, "parsing claim size %s", claimSize)
+
+	pvc := &corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      pvcName,
+			Namespace: f.Namespace.Name,
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			StorageClassName: &scName,
+			AccessModes: []corev1.PersistentVolumeAccessMode{
+				corev1.ReadWriteOnce,
+			},
+			Resources: corev1.VolumeResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceStorage: claimQuantity,
+				},
+			},
+		},
+	}
+	pvc, err = f.ClientSet.CoreV1().PersistentVolumeClaims(f.Namespace.Name).Create(ctx, pvc, metav1.CreateOptions{})
+	e2e.ExpectNoError(err, "creating PVC %s", pvcName)
+
+	g.DeferCleanup(func(ctx context.Context) {
+		err := f.ClientSet.CoreV1().PersistentVolumeClaims(f.Namespace.Name).Delete(ctx, pvc.Name, metav1.DeleteOptions{})
+		e2e.ExpectNoError(err, "deleting PVC %s", pvc.Name)
+	})
+
+	podConfig := &e2epod.Config{
+		NS:            f.Namespace.Name,
+		PVCs:          []*corev1.PersistentVolumeClaim{pvc},
+		NodeSelection: e2epod.NodeSelection{Name: nodeName},
+		Command:       "ls -la " + e2epod.VolumeMountPath1,
+	}
+	pod, err := e2epod.MakeSecPod(podConfig)
+	// Make the pod name nicer to users (it has random uuid otherwise)
+	pod.Name = fmt.Sprintf("pod-%d", podNumber)
+	e2e.ExpectNoError(err, "preparing pod %d", podNumber)
+	err = e2epod.WaitForPodNotFoundInNamespace(ctx, f.ClientSet, pod.Name, pod.Namespace, f.Timeouts.PodDelete)
```

Comment:
Why this particular line before pod creation? Also, we are ignoring the returned error.

Reply:
This is some leftover, removed.